radis/api/kuruczapi.py

```diff
@@ -0,0 +1,551 @@
+        eupper_inverted = np.where((eupper - elower) > 0, eupper, elower)
+        jlower_inverted = np.where((eupper - elower) > 0, jlower, jupper)
+        jupper_inverted = np.where((eupper - elower) > 0, jupper, jlower)
+        elower = elower_inverted
+        eupper = eupper_inverted
+        jlower = jlower_inverted
+        jupper = jupper_inverted
+
+        wlaa = np.where(wlnmair < 200, wlnmair * 10, air2vacuum(wlnmair * 10))
+        nu_lines = 1e8 / wlaa[::-1]  # [cm-1]<-[AA]
+        loggf = loggf[::-1]
+        ielem = ielem[::-1]
+        iion = iion[::-1]
+        elower = elower[::-1]
+        eupper = eupper[::-1]
+        jlower = jlower[::-1]
+        jupper = jupper[::-1]
+        gupper = jupper * 2 + 1
+        A = (
+            10**loggf
+            / gupper
+            * (self.ccgs * nu_lines) ** 2
+            * (8 * np.pi**2 * self.ecgs**2)
+            / (self.mecgs * self.ccgs**3)
+        )
+        gamRad = gamRad[::-1]
+        gamSta = gamSta[::-1]
+        gamvdW = gamvdW[::-1]
+
+        data_dict = {
+            "A": A,
+            "nu_lines": nu_lines,
+            "wav": 10**7/nu_lines,
+            "El": elower,
+            "eupper": eupper,
+            "gu": gupper,
+            "jlower": jlower,
+            "ju": jupper,
+            "id": ielem,
+            "iso": iion,
+            "gamRad": gamRad,
+            "gamSta": gamSta,
+            "gamvdW": gamvdW,
+            "Tdpair": 0.68, #placeholder to adjust
+            "wlnmair": wlnmair,
+            "loggf": loggf,
+            "species": species,
+            "labellower": labellower,
+            "labelupper": labelupper,
+            "ref": ref,
+            "NLTElower": NLTElower,
+            "NLTEupper": NLTEupper,
+            "isonum": isonum,
+            "hyperfrac": hyperfrac,
+            "isonumdi": isonumdi,
+            "isofrac": isofrac,
+            "hypershiftlower": hypershiftlower,
+            "hypershiftupper": hypershiftupper,
+            "hyperFlower": hyperFlower,
+            "hypernotelower": hypernotelower,
+            "hyperFupper": hyperFupper,
+            "hypternoteupper": hypternoteupper,
+            "strenclass": strenclass,
+            "auto": auto,
+            "landeglower": landeglower,
+            "landegupper": landegupper,
+            "isoshiftmA": isoshiftmA,
+            #"int":self.Sij0(A,gupper,nu_lines,elower,self.partfcn(self.key,296))
+            #if int parameter is used, calc_linestrength_eq will also use it
+        }
+
+        df_radis = pd.DataFrame(data_dict)
+
+        self.data = pd.DataFrame(data_dict)
+        return df_radis
+    
+    def add_airbrd(self,data):
+        if "airbrd" not in data.columns :
+            #placeholder for neutral_hydrogen_number and atomic_coeff
+            #TODO: adjust the coefficient to the atoms and adjust the value for neutral_hydrogen_number
+            
+            neutral_hydrogen_number=1
+            atomic_coeff=1
+
```

Comment:
@minouHub The other placeholders for atomic_coeff and neutral_hydrogen_number lines 399 and 400